src/main/java/org/takes/rs/RsPrettyXML.java

```diff
@@ -165,11 +165,15 @@ private static void prepareDocType(final InputStream body,
                     && "html".equalsIgnoreCase(doctype.getName())) {
                     transformer.setOutputProperty(OutputKeys.METHOD, "html");
                     transformer.setOutputProperty(OutputKeys.VERSION, "5.0");
-                } else {
+                    return;
```

Comment:
Look at  my 2nd commit. This part of code already refactored according to checkstyle rules.